src/PrestaShopBundle/Controller/Admin/Improve/Modules/ModuleAbstractController.php

```diff
@@ -59,12 +59,17 @@ final protected function getNotificationPageData(ModuleCollection $moduleCollect
         ];
     }
 
+    protected function getModuleRepository(): ModuleRepositoryInterface
+    {
+        return $this->get('prestashop.core.admin.module.repository');
+    }
+
     /**
      * Common method for all module related controller for getting the header buttons.
      *
      * @return array
      */
-    protected function getToolbarButtons()
+    protected function getToolbarButtons(): array
```

Comment:
I'm not sure you should change this signature as it would cause a breaking change for existing controllers that extend this class and won't have the strict type